dungeon-crawler/game_engine/src/main/java/com/novoda/dungeoncrawler/EnemySpawner.java

```diff
@@ -1,6 +1,6 @@
 package com.novoda.dungeoncrawler;
 
-class EnemySpawner {
+public class EnemySpawner {
```

Comment:
hmmmmmmm

Reply:
kotlin needs all these to be public 😓 should be "fixed" once it's all kotlin 😄